src/components/post/Post.tsx

```diff
@@ -0,0 +1,37 @@
+import {createContext} from 'react'
+import './Post.css'
+import PostBody from './PostBody/PostBody';
+import PostInfoBar from './PostInfoBar/PostInfoBar';
+import PostReactions from './PostReactions/PostReactions';
+
+interface PostPropsTypes {
+    text: string;
+    author: string;
+    date: Date;
+    likesAmount: number;
+    comments: string[];
+}
+
+export const PostContext = createContext({text: "", author: "", date: new Date(), likesAmount: 0, 
```

Comment:
give type to your context:
```suggestion
export const PostContext = createContext<PostPropsTypes>({text: "", author: "", date: new Date(), likesAmount: 0, 
```